apis/fluentbit/v1alpha2/clusterfluentbitconfig_types.go

```diff
@@ -46,6 +47,10 @@ type FluentBitConfigSpec struct {
 	// If namespace is defined, then the configmap and secret for fluent-bit is in this namespace.
 	// If it is not defined, it is in the namespace of the fluentd-operator
 	Namespace *string `json:"namespace,omitempty"`
+	// ConfigFileFormat defines the format of the config file, default is "classic",
+	// available options are "classic" and "yaml"
+	// +kubebuilder:validation:Enum:=classic;yaml
+	ConfigFileFormat *string `json:"configFileFormat,omitempty"`
```

Comment:
What would be the default configuration format?

Reply:
the default will be the classic mode.